### PR TITLE
Add the grant funding to the claim check page and update pages to match prototype 

### PR DESCRIPTION
**Pull request by gms-gs**
Add the grant funding to the claim check page and update pages to match prototype

## Context

Update pages to show claim amount and also update content

## Changes proposed in this pull request

Update UI for various pages

## Guidance to review

- View the pages which have changed 

## Link to Trello card

https://trello.com/c/R4wX1blP/208-add-the-grant-funding-to-the-claim-check-page

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated or [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

File: app/views/claims/schools/claims/check.html.erb [hunk 1]
```diff
@@ -15,7 +15,12 @@
 
       <%= govuk_summary_list do |summary_list| %>
         <% summary_list.with_row do |row| %>
-          <% row.with_key(text: Claim.human_attribute_name(:provider)) %>
+          <% row.with_key(text: t(".school_name")) %>
```
Comment (CatalinVoineag) [inline app/views/claims/schools/claims/check.html.erb:18]: Can we do?

```suggestion
          <% row.with_key(text: Claim.human_attribute_name(:school_name)) %>
```

File: app/models/region.rb [hunk 2]
```diff
@@ -20,4 +20,8 @@ class Region < ApplicationRecord
   validates :name, presence: true, uniqueness: { case_sensitive: false }
   validates :claims_funding_available_per_hour_currency, presence: true
   validates :claims_funding_available_per_hour_pence, presence: true
+
+  def funding_available_per_hour
```
Comment (CatalinVoineag) [inline app/models/region.rb:24]: Can we add a spec for this?

**Review by CatalinVoineag — COMMENTED** [review]
Minor things

**Review by CatalinVoineag — APPROVED** [review]
:shipit:

**Review by JamieCleare2525 — APPROVED** [review]

**Pull request merged**